angular-app/src/app/workspace/workspace.component.ts

```diff
@@ -73,6 +76,18 @@ export class WorkspaceComponent implements OnInit {
       });
     }
 
+
+    saveLyrics() {
+        var lyricDoc = (document.getElementById("lyric-doc") as HTMLInputElement);
+        this.projectService.updateLyricDoc(this.currProjId, lyricDoc.value);
+    }
+
+    getLatestLyrics(project) {
```

Comment:
What if user enters some text in the lyrics text box, and then hits reload? Would we want to display some kind of prompt?

Reply:
After trying it for a little, the reload button will no work unless the textbox is already empty. When the textbox is empty and the user does not hit reload but save instead, the previous save is overwritten. Is there any way that I can make the reload function start when the page is loaded?

Reply:
If you haven't already, it's worth trying to run the function in the ngOnInit function to reload on start.